lib/bolt/cli.rb

```diff
@@ -98,16 +98,18 @@ def initialize(argv)
     def create_option_parser(results)
       OptionParser.new('') do |opts|
         opts.on(
-          '-n', '--nodes x,y,z', Array,
+          '-n', '--nodes NODES',
           'Node(s) to connect to in URI format [protocol://]host[:port]',
           'Eg. --nodes bolt.puppet.com',
           'Eg. --nodes localhost,ssh://nix.com:2222,winrm://windows.puppet.com',
           "\n",
+          '* NODES can either be comma-separated, \'@<file>\' to read',
```

Comment:
`@` has special meaning in PowerShell, so I would prefer to avoid it here if possible.

Reply:
We have the same issue with parameters, e.g. `--params @params.json`. I was going to keep this consistent, and fix both in the other ticket.